.github/workflows/build_check.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Build Check
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
not just master, should have this build check **on every commit to every branch**